actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -80,12 +78,22 @@ def format(formatter, filter = nil)
         formatter.result
       end
 
+      def self.defaults_filter_for(filter)
+        {
+            controller: /#{filter}/,
```

Comment:
We know a controller must adhere to a certain format. So let's:

- Move the normalization to here (downcase etc.)
- Rename to `normalize_filter`
- Turn into private instance method
- Make sure action isn't used when passing `-g`